### PR TITLE
AR: inserted David Johnson exception

### PR DESCRIPTION
**Pull request by linzjax**
He didn't have a party tag next to his name. Created an exception that would break if he added one later.

### CODE REVIEW

File: openstates/ar/legislators.py [hunk 1]
```diff
@@ -48,7 +48,13 @@ def scrape_member(self, chamber, term, member_url):
 
         party = name_and_party[-1]
 
-        if party == '(R)':
+        if ' '.join(name_and_party[1:]) == 'David Johnson' and "(" not in party:
+                full_name = ' '.join(name_and_party[1:])
+                party = 'Democratic'
+        elif ' '.join(name_and_party[1:]) == 'David Johnson' and "(" in party:
```
Comment (mileswwatkins) [inline openstates/ar/legislators.py]: If a party _does_ come back, then `' '.join(name_and_party[1:]) == 'David Johnson'` will be false, right?

**Comment by mileswwatkins** [comment]
Also, can you add code to make sure that an error will be thrown if David Johnson is no longer in the legislature? This way the obsolete code can be thrown out when it's no longer needed.

**Comment by mileswwatkins** [comment]
Otherwise, looks great to me!

**Comment by linzjax** [comment]
Alright, seems to be working.

File: openstates/ar/legislators.py [hunk 2]
```diff
@@ -48,7 +53,13 @@ def scrape_member(self, chamber, term, member_url):
 
         party = name_and_party[-1]
 
-        if party == '(R)':
+        if ' '.join(name_and_party[1:]) == 'David Johnson' and "(" not in party:
+            full_name = ' '.join(name_and_party[1:])
+            party = 'Democratic'
+            global _Remove_special_casing
```
Comment (mileswwatkins) [inline openstates/ar/legislators.py]: You don't need to declare a `global` variable; it already exists since you declared it on line 8.

**Pull request merged**